pages/templates/pages/add_group.html

```diff
@@ -224,7 +224,8 @@ <h4 class="mb0">{% bootstrap_label "Regular Meeting" label_for="recurring_meetin
           </div>
         </div>
 
-      <input type="hidden" id="id_group_id" name="group_id" value='9999'>
+        <input type="hidden" id="id_group_id" name="group_id" value='9999'>
+        <input type="hidden" id="id_group_type" name="group_type" value='local-group'>
```

Comment:
I suggest we remove these hidden fields. This is kind of an anti-pattern - aren't we basically trying to say these fields don't belong on this form because the user is not really supplying these values? We are going to set these values later so they should not appear in the markup. Also users can edit markup in browser before submitting, so this is not a reliable approach.

Reply:
This is a required value, so unless we set blank = true, this is required on all forms. If we do set blank = true, we'll get an error on admin side if a group_type is saved as none because null = false. If we set null = false, we'll allow null in these fields. So this was only solution that made sense.

Reply:
Disregard found a solution.

Reply:
I think there is a way to separate the form logic from the model logic. The model stored in db has certain fields that are required, but if they are generated automatically they shouldn't be on the form. For example `ID` and `date_created` fields are usually generated automatically, and don't appear on a form. So there needs to be some separation of logic in some cases between forms and models. I don't think putting them as hidden input fields in html with invalid data values is a good solution, and I suggest we avoid this.

Reply:
I understand we have a lot of legacy code and tech debt etc., but still want to establish what is our standard here, and I think it should be decoupling form logic from model logic when needed in cases like this.